### ISSUE TITLE
Please remove the post error before captcha feature and add a cooldowm timer

### ISSUE THREAD

**Issue by twozerozero** [Adamantcheese/Kuroba#181]
4chanX has a similar feature. You fill in the captcha, then press the send button (where "send" is replaced by the number of seconds left in the cooldown), then when the captcha gets accepted it auto posts when the cooldown is done.
For Kuroba, it could work where there is a timer you can tap (that shows up where the send button is) and it lets you fill out the captcha. Once the captcha is done it sends you back to the post screen and it automatically posts the message when the cooldown timer runs out

**Comment by Adamantcheese** [comment]
Duplicate of #15.
